test/spec/geo/leaflet/leaflet-tiled-layer-view.spec.js

```diff
@@ -2,70 +2,132 @@ var Backbone = require('backbone');
 var LeafletTiledLayerView = require('../../../../src/geo/leaflet/leaflet-tiled-layer-view');
 
 describe('leaflet-tiled-layer-view', function () {
-  beforeEach(function () {
-    this.layerModel = new Backbone.Model({
-      urlTemplate: 'http://whatever.com',
-      subdomains: 'a',
-      opacity: 1,
-      attribution: 'CARTO',
-      errorTileUrl: 'http://matall.in',
-      maxZoom: 10,
-      minZoom: 0,
-      tms: false
+  describe('does not support high resolution screens', function () {
```

Comment:
(Another personal taste thing)
I think most of the tests in cartodb.js follow the [given, when, then](https://martinfowler.com/bliki/GivenWhenThen.html) rule so they are easier to follow.


```
(given a) 
leaflet-tiled-layer-view
    when high-resolution screens are not supported
        should have options set correctly
```